share/misc/bsd-family-tree

```diff
@@ -447,39 +447,45 @@ FreeBSD 5.2           |      |                 |                       |
  |  FreeBSD           |      |                    |                    |
  |   13.2             |      |                    |                    |
  |                    |      |                    |                    |
-FreeBSD 14 -current   |  NetBSD -current   OpenBSD -current    DragonFly -current
+ *--FreeBSD           |      |                    |                    |
+ |   14.0 prerelease  |      |                    |                    |
+ |                    |      |                    |                    |
+FreeBSD 15.0-CURRENT  |  NetBSD-current  OpenBSD-current  DragonFly-DEVELOPMENT
  |                    |      |                    |                    |
  v                    v      v                    v                    v
 
 Time
 ----------------
 
-Time tolerance +/- 6 months, depending on which book/article you read; if it
-was the announcement in Usenet or if it was available as tape.
+Time tolerance on 4.3BSD and earlier releases can be as large as six months due
+to the uncertainty of historical records; the early release to special groups;
+and the then slowness of tape duplication. Articles and books from the period
```

Comment:
You can omit 'then' here. It reads better without it.

Reply:
Resolved by [`a7017bb` (#812)](https://github.com/freebsd/freebsd-src/pull/812/commits/a7017bbfdacb995bcccfd6a42e436671591a7307).